test-suite/misc/coqtop_print-mod-uid.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+export COQBIN=$BIN
+export PATH=$COQBIN:$PATH
+
+function assert_eq() {
+  if [ "$1" != "$2" ]; then
+    echo "Unexpected output: " $1 "!=" $2
+    cd /
+    rm -rf $TMP
```

Comment:
I am always a bit worried I see `rm -rf` just next to `cd /`. Anyway, this can all be removed now.